client.py

```diff
@@ -297,6 +297,8 @@ async def on_ready():
 
         asyncio.ensure_future(loop_utils.decrease_food_multiplier())
 
+
+        asyncio.ensure_future(loop_utils.drunk_effects(id_server=server.id))
```

Comment:
If inebriation does something fr fr, you should change most of the fuck energies to not have comically-high inebriation values. Set most to 0, except, like, krakacola and goonshine.